web-client/src/ducks/requests/actions.ts

```diff
@@ -1,39 +1,60 @@
 import { IRequest, Request } from 'src/models/requests';
```

Comment:
Renamed OFFER to CAV_REQUEST and REQUEST to PIN_REQUEST.  Created separate function for FND as the logic for FIND is different and it is confusing having them both in the same file.